install.sh

```diff
@@ -279,60 +271,58 @@ for i in $(seq 0 $((${#urls[@]} - 1))); do
   tarfile=$(basename "${url}")
   name="${tarfile%%-*}"   # extract string before first '-'
   rest="${tarfile#*-}"    # extract string after first '-'
-  version="${rest%%-chromeos*}"
-                          # extract string between first '-' and "-chromeos"
+  version="${rest%%-chromeos*}" # extract string between first '-' and "-chromeos"
 
   download_check "${name}" "${url}" "${tarfile}" "${sha256}"
   extract_install "${name}" "${tarfile}"
   update_device_json "${name}" "${version}"
 done
 
 echo_info "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
-echo -e "${GRAY}"
+
+# Work around https://github.com/chromebrew/chromebrew/issues/3305.
+sudo ldconfig &> /dev/null || true
+echo_out "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
 ln -sfv "../lib/crew/bin/crew" "${CREW_PREFIX}/bin/"
-echo -e "${RESET}"
 
-echo_info "Setup and synchronize local package repo..."
-echo -e "${GRAY}"
+echo_out "Set up and synchronize local package repo..."
 
-# Remove old git config directories if they exist.
-find "${CREW_LIB_PATH}" -mindepth 1 -delete
+# Set LD_LIBRARY_PATH so crew doesn't break on i686 and
+# the mandb postinstall doesn't fail.
+echo "export LD_LIBRARY_PATH=$CREW_PREFIX/lib$LIB_SUFFIX" >> $CREW_PREFIX/etc/env.d/path
```

Comment:
Should we rename `path` to `00-path` so it gets exported first?  Also, the `export` is not needed.

Reply:
`00-path` would be overwritten by `crew_profile_base`. Actually, wouldnt it make more sense to have these variables set [here](https://github.com/chromebrew/crew-profile-base/blob/master/src/env.d/00-path)?

Reply:
Yes, that is what I meant.  Probably should have said append to the existing 00-path.  Keep in mind this is only necessary if `LD_LIBRARY_PATH` is not equal to `/usr/local/lib` or `/usr/local/lib64` since that is already the default setting on a freshly powerwashed system.

Reply:
Any updates to `crew_profile_base` will overwrite that appending, though.

>  Keep in mind this is only necessary if LD_LIBRARY_PATH is not equal to /usr/local/lib or /usr/local/lib64 since that is already the default setting on a freshly powerwashed system.

It's the logic that was already in the installer, and satmandu set it back in 7436. I can't test if i686 breaks without it, but I presume it did otherwise that line wouldn't have been added.

Reply:
I tested it on an i686 container without this line, and it worked. @satmandu do you get the same result? We might need to test on real hardware here, and I don't have an i686 chromebook.